openwisp_controller/geo/admin.py

```diff
@@ -130,14 +131,36 @@ class DeviceLocationFilter(admin.SimpleListFilter):
     title = _("has geographic position set?")
     parameter_name = "with_geo"
 
+    def __init__(self, request, params, model, model_admin):
+        super().__init__(request, params, model, model_admin)
+        if config_app_settings.WHOIS_CONFIGURED:
+            self.title = _("geographic position")
+
     def lookups(self, request, model_admin):
+        if config_app_settings.WHOIS_CONFIGURED:
+            return (
+                ("outdoor", _("Outdoor (Excluding Estimated)")),
+                ("indoor", _("Indoor")),
+                ("estimated", _("Estimated")),
+                ("false", _("No Location")),
+            )
         return (
             ("true", _("Yes")),
             ("false", _("No")),
         )
 
     def queryset(self, request, queryset):
-        if self.value():
+        if value := self.value():
```

Comment:
```suggestion
        value = self.value()
		if not value:
    		return queryset
```

let's early return from here

Reply:
I think the current logic also works the same way since there are return statements for each flow.
Nonetheless, having early return makes it a bit more readable and safe guard

Reply:
yes this comment was only for readability